resources/css/style.css

```diff
@@ -1056,7 +1056,7 @@ i.bx {
 }
 .notifacion-card > ul li {
   display: flex;
-  align-items: flex-start;
+  align-items: start;
```

Comment:
The browser swears at align-items: start. It should be done through flex-start/flex-end